content/en/calendars/_index.md

```diff
@@ -2,6 +2,4 @@
 title: Community Calendars 📅
 ---
 
-These are calendars for projects in the Scientific Python community.
-You can subscribe to these calendars from any calender software that
-supports ICAL.
+Subscribe from any calender software that supports ICAL by **copying** the following URLs:
```

Comment:
This is not quite right: to enter events into calendar software that supports ICAL, you will need to download the file.  But, for Google Calendar and others that support refreshing from a URL, you can copy the ICAL URL and they will re-fetch every 12 hours or so.